lightly/models/modules/__init__.py

```diff
@@ -17,3 +17,8 @@
 from lightly.models.modules.heads import SimSiamProjectionHead
 from lightly.models.modules.heads import SimSiamPredictionHead
 from lightly.models.modules.nn_memory_bank import NNMemoryBankModule
+from lightly.models.modules.momentum import batch_shuffle
```

Comment:
None of these are modules, they are functions. I would not put them here. Maybe we can have something like `lightly.functionals` or put them under `lightly.utils`.

Reply:
After the discussion with @IgorSusmelj we have another idea: Put it in `lightly.models.utils(.py)` or put it in `lightly.models.momentum.py`